dubbo-metrics/dubbo-metrics-default/src/main/java/org/apache/dubbo/metrics/collector/sample/ThreadPoolMetricsSampler.java

```diff
@@ -72,13 +73,12 @@ public void addExecutors(String name, ExecutorService executorService) {
 
     @Override
     public List<MetricSample> sample() {
-        List<MetricSample> metricSamples = new ArrayList<>();
-
-        sampleThreadPoolExecutor.forEach((name, executor) -> {
-            metricSamples.addAll(createMetricsSample(name, executor));
-        });
+        return sampleThreadPoolExecutor.entrySet()
+            .stream()
+            .map(ele -> createMetricsSample(ele.getKey(), ele.getValue()))
+            .flatMap(Collection::stream)
+            .collect(Collectors.toList());
 
```

Comment:
A map can be omitted. sampleThreadPoolExecutor.entrySet().stream()
             .flatMap(entry -> createMetricsSample(entry.getKey(), entry.getValue()).stream())
             .collect(Collectors.toList());

Reply:
ok, and it already fixed